### PR TITLE
WRO-13139: Sampler: Remove controls from Docs tab

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
In packages/sampler storybook, if controls are changed from Docs tab, it doesn't update the component preview.

### Resolution
Setting "inlineStories" to true causes bugs, so we agreed to keep inlineStories false and set a height to the iFrame. Also, controls were removed from the docs page.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-13139

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM but let's see SWP team opinion

File: packages/sampler/.storybook/preview.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import {configureActions} from '@enact/storybook-utils/addons/actions';
 import {getBooleanType, getObjectType} from '@enact/storybook-utils/addons/controls';
-import {DocsPage, DocsContainer} from '@enact/storybook-utils/addons/docs';
+import {DocsContainer, Primary, Title} from '@enact/storybook-utils/addons/docs';
```
Comment (MikyungKim) [inline packages/sampler/.storybook/preview.js:3]: We need to update storybook-utils version to make this work.
We'll release storybook-utils soon so let's update the PR after release.

File: packages/sampler/.storybook/preview.js [hunk 2]
```diff
@@ -37,12 +38,18 @@ const backgrounds = {
...
 
 configureActions();
+
 export const parameters = {
 	docs: {
 		container: DocsContainer,
-		page: DocsPage,
-		iframeHeight: 360,
 		inlineStories: false,
+		iframeHeight: ri.scaleToRem(900),
```
Comment (MikyungKim) [inline packages/sampler/.storybook/preview.js:46]: The height seems too big. 300 was alright.

File: packages/sampler/.storybook/preview.js (hunk 1 above)
Comment (MikyungKim) [reply packages/sampler/.storybook/preview.js:3]: We've released storybook-utils 5.0.1. Please update the PRs.

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**